src/cli/main.c

```diff
@@ -458,8 +458,7 @@ int main(int argc, char *arg[])
       g_free(output_filename);
     if(output_ext)
```

Comment:
Those are unrelated changes, and not related to coding style. cppcheck identifed `if(inputs)` to be a redundant condition as it is already checked previously. Not so for `if(output_ext)` and `if(output_filename)`, although I agree they are somewhat unnecessary as well since g_free()/free() will work on NULL pointers.